### ISSUE TITLE
Add "send test email" feature for self-hosted installations

### ISSUE THREAD

**Issue by taobojlen** [plausible/analytics#361]
## Prerequisites
- [x] I have searched open and closed issues to make sure that the feature has not yet been requested.

## Feature request
Email reports on my self-hosted Plausible instance don't seem to be working, and I'm having some trouble debugging it: I can't find any Oban jobs from the `Plausible.Workers.SendEmailReports` module, and the functions that actually send the email are private in the workers. It'd be nice if there was some way to send a test email and display potential errors.

**How would you like it to work**
A button in the settings would let you send a test email to yourself (if your address has been verified). If any errors occur, they can be displayed in the front-end.

Alternatively, there could be a way to send such an email from the IEX console.

**Comment by Maronato** [comment]
It [seems](https://github.com/plausible/analytics/issues/400) that setting the environment var `CRON_ENABLED` to `true` is required for email reports to be sent.

A way to test email delivery is to request a password change from the login screen.

**Comment by ukutaht** [comment]
Thanks @Maronato 

I will have to clarify the docs and config options. As for this issue, you can test email delivery either by signing up with a new email address (activation email) or resetting password if you already have an account.

**Comment by ndbroadbent** [comment]
Sorry to comment on an old thread, but I was also having email issues. Are the scheduled email reports still disabled by default? It looks like `CRON_ENABLED` is still set to `false` by default. I've used the password reset feature to confirm that emails are working (with Postmark), and I've just set `CRON_ENABLED=true`, so hopefully will see some email reports soon.

<img width="753" alt="Screen Shot 2022-10-13 at 8 21 33 PM" src="https://user-images.githubusercontent.com/139536/195529025-21f0d1bf-e70f-4804-8171-32a292de49d7.png">
